shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2462,6 +2466,23 @@ - (void)clearTextInputClient {
   _activeView.frame = CGRectZero;
 }
 
+- (void)updateConfig:(NSDictionary*)dictionary {
+  BOOL isSecureTextEntry = [dictionary[kSecureTextEntry] boolValue];
+  for (UIView* view in self.textInputViews) {
+    if ([view isKindOfClass:[FlutterTextInputView class]]) {
+      FlutterTextInputView* inputView = (FlutterTextInputView*)view;
+      // The feature of holding and draging spacebar to move cursor is affected by
+      // secureTextEntry, so when obscureText is updated, we need to update secureTextEntry
+      // and call reloadInputViews.
+      // https://github.com/flutter/flutter/issues/122139
+      if (inputView.isSecureTextEntry != isSecureTextEntry) {
+        inputView.secureTextEntry = isSecureTextEntry;
+        [inputView reloadInputViews];
```

Comment:
The document of `reloadInputViews` this method says:
```You can use this method to refresh the custom input view or input accessory view associated with the current object when it’s the first responder. The views are replaced immediately, without animating them into place. If the current object isn’t the first responder, this method has no effect.``` 
(https://developer.apple.com/documentation/uikit/uiresponder/1621110-reloadinputviews?language=objc)

"The views are replaced immediately" sounds like the some view inside the `inputView` is replaced? What are other side effects of replacing the view? Do we not need to restore any state of the view?

Reply:
I didn't find any relevant side effect descriptions, and there was no problem after testing, so I think that the `reloadInputViews` method simply refreshes the keyboard view and does not cause other side effects. And when switching `secureTextEntry` to `true`, the keyboard will not have input accessory view, so there is no need to save the state.

In addition, I did a test in a native project, calling the `reloadInputViews` method directly will not affect the state in input accessory view.

https://user-images.githubusercontent.com/19367531/227701053-dd586326-9e33-41ad-b680-ea9f50d5db30.mov